x-pack/plugins/apm/common/utils/param_query.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { escapeKuery } from '@kbn/es-query';
+
+export function paramQuery<T extends string>(
+  field: T,
+  value: string | boolean | number | undefined | null
+) {
+  if (!value) return [];
+
+  const escapedValue = escapeKuery(value.toString());
+
+  return [`${[field]}: ${escapedValue}`];
+}
```

Comment:
The name makes it sound like it's an ES query, but it convers a field/value pair to a KQL expression. Can you use a name that reflects that?

Reply:
good point. I changed it to `fieldValuePairToKql`